pages/probes/[id].vue

```diff
@@ -1,414 +1,361 @@
 
-	// ACTIONS
+		probeDetailsUpdating.value = true;
 
-	const isSaveEnabled = computed(() => (
-		probe.value.name !== props.probe.name
-		|| probe.value.country !== props.probe.country
-		|| probe.value.city !== props.probe.city
-		|| !isEqual(probe.value.tags, props.probe.tags)
-		|| isEditingTags.value
-	));
+		if (!probeDetails.value) {
+			probeDetailsUpdating.value = false;
 
+			return;
+		}
 
-	const updateProbeLoading = ref(false);
-	const updateProbe = async () => {
-		updateProbeLoading.value = true;
-		const tags = isEditingTags.value ? convertTags(tagsToEdit.value) : probe.value.tags;
+		if (editedName.value === originalName.value) {
+			isEditingName.value = false;
+			probeDetailsUpdating.value = false;
 
-		if (areTagsEmpty(tags)) {
-			updateProbeLoading.value = false;
-			sendToast('error', 'Tags are invalid', 'Some tag values are empty');
 			return;
 		}
 
 		try {
-			await $directus.request(updateItem('gp_probes', probe.value.id, {
-				...(probe.value.name !== props.probe.name && { name: probe.value.name }),
-				...(probe.value.country !== props.probe.country && { country: probe.value.country }),
-				...(probe.value.city !== props.probe.city && { city: probe.value.city }),
-				...(!isEqual(tags, props.probe.tags) && { tags }),
-			}));
+			await $directus.request(updateItem('gp_probes', probeDetails.value.id, { name: editedName.value }));
 
 			sendToast('success', 'Done', 'The probe has been successfully updated');
 			emit('save');
-			probeDetailsDialog.value = false;
+
+			originalName.value = editedName.value;
+			isEditingName.value = false;
+			probeDetails.value.name = editedName.value;
 		} catch (e) {
 			sendErrorToast(e);
+
+			if (inputNameRef.value) {
+				inputNameRef.value.focus();
+			}
+		} finally {
+			probeDetailsUpdating.value = false;
+		}
+	};
+
+	// HANDLE PRIMARY, ALT IPS
+	const limitIpsToShow = () => {
+		if (showMoreIps.value) {
+			return probeDetails?.value?.altIps;
 		}
 
-		updateProbeLoading.value = false;
+		return probeDetails?.value?.altIps?.slice(0, 1) ?? [];
 	};
 
-	const deleteDialog = ref(false);
-	const deleteProbeLoading = ref(false);
-	const deleteProbe = async () => {
-		deleteProbeLoading.value = true;
 
-		try {
-			await $directus.request(updateItem('gp_probes', probe.value.id, {
-				userId: null,
-			}));
+	// HANDLE STATUS COLORS
+	const getProbeStatusColor = (status: string) => {
+		switch (status.toLowerCase()) {
+		case 'ready':
+			return 'text-green-500';
 
-			sendToast('success', 'Done', 'The probe has been deleted');
-			emit('delete');
-			probeDetailsDialog.value = false;
-		} catch (e) {
-			sendErrorToast(e);
+		case 'offline':
+			return 'text-bluegray-500';
+
+		case 'ping-test-failed':
+			return 'text-red-500';
+
+		default:
+			return 'text-yellow-600';
 		}
+	};
 
-		deleteProbeLoading.value = false;
+	// HANDLE CREDITS
+	const probeCreditsPerMonth = ref<number | null>(null);
+
+	// TODO: update loadCreditsData once BE is ready
```

Comment:
After that PR is merged, you can filter by `adopted_probe` field:
https://github.com/jsdelivr/globalping-dash-directus/pull/81